msrestazure/azure_active_directory.py

```diff
@@ -525,3 +524,81 @@ def set_token(self, response_url):
             raise_with_traceback(AuthenticationError, "", err)
         else:
             self.token = token
+
+
+class AdalAuthentication(Authentication):  # pylint: disable=too-few-public-methods
+    """A wrapper to use ADAL for Python easily to authenticate on Azure.
```

Comment:
Put the closing `"""` on the same line and have a newline between it and `__init__()`.